bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/types/DateTimeType.java

```diff
@@ -148,9 +148,7 @@ public String format(Locale locale, String pattern) {
      * Create a {@link DateTimeType} being the translation of the current object to the locale time zone
      *
      * @return a {@link DateTimeType} translated to the locale time zone
-     *
      * @throws DateTimeException if the converted zone ID has an invalid format or the result exceeds the supported date
-     *             range

```

Comment:
May I ask you to readd this line (see [L163](https://github.com/openhab/openhab-core/pull/1109/files#diff-eb3af4acb8aaa05b678d2d699ee585beL167])).

Reply:
Fixed!